src/main/java/com/nashtech/exception/WebClientCustomException.java

```diff
@@ -0,0 +1,32 @@
+package com.nashtech.vehicleapplication.exception;
+
+/**
+ * Exception class to represent errors that
+ * occur during WebClient operations.
+ */
+public class WebClientCustomException extends
```

Comment:
exception class should be related to the error. No need to add `Custom`. You can take like MockarooException

Reply:
Made the suggested changes